src/backend/_scss/application.scss

```diff
@@ -659,10 +659,47 @@ a {
 
 .track-names {
   color: $black;
-  margin-top: 16%;
+  margin-top: 0;
+}
+
+.social-buttons {
```

Comment:
@aayusharora mixin is good idea but the tag at `665` and `712` are not repeated code.